docs/agents/models.md

```diff
@@ -27,7 +27,73 @@ ADK primarily uses two mechanisms for model integration:
 
 The following sections guide you through using these methods based on your needs.
 
-## Using Google Gemini Models
+## Google Gemini models
+
+ADK support the Google Gemini family of generative AI models that provide a powerful
+set of models with a wide range of features. ADK provides a number of
+integrations with Gemini models to support features including
+[Code Execution](/adk-docs/tools/built-in-tools/#code-execution),
+[Google Search](/adk-docs/tools/built-in-tools/#google-search),
+[Context caching](/adk-docs/context/caching/),
+[Computer use](/adk-docs/tools/gemini-api/computer-use/)
+and the [Interactions API](#interactions-api).
+
+### Gemini Interactions API {#interactions-api}
+
+<div class="language-support-tag" title="Java ADK currently supports Gemini and Anthropic models.">
+  <span class="lst-supported">Supported in ADK</span><span class="lst-python">Python v1.21.0</span>
+</div>
+
+The Gemini [Interactions API](https://ai.google.dev/gemini-api/docs/interactions)
+provides stateful conversation capabilities, allowing you to chain interactions
```

Comment:
Is an alternative to the generateContent inference API, which provides stateful conversation capabilities, allowing you to chain interactions...

Reply:
thanks. it's a useful detail to call out the existing generateContent()